client/src/pages/private/ParticipantTable.js

```diff
@@ -400,43 +401,23 @@ export default () => {
       await getParticipants();
 
       setColumns((oldColumns) => {
-        setHideLastNameAndEmailFilter(
-          ['Available Participants', 'Archived Candidates'].includes(reducerState.tabValue)
-        );
-
-        if (reducerState.tabValue !== 'Available Participants')
-          oldColumns = oldColumns.filter((column) => column.id !== 'callbackStatus');
-
-        if (
-          ['My Candidates', 'Archived Candidates'].includes(reducerState.tabValue) &&
-          !oldColumns.find((column) => column.id === 'status')
-        ) {
-          // Remove statusInfo colum
-          oldColumns = oldColumns.filter((column) => column.id !== 'statusInfo');
-          return [
-            ...oldColumns.slice(0, 3),
-            { id: 'status', name: 'Status' },
-            ...oldColumns.slice(3),
-          ];
+        switch (reducerState.tabValue) {
```

Comment:
This looks like it's going to conflict pretty heavily with HCAP-660, we should discuss